<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterialTypeWidget</class>
 <widget class="QWidget" name="AddMaterialTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parent material :</string>
   </property>
  </widget>
  <widget class="TreeBox" name="parentMaterialComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>40</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name* :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeBox</class>
   <extends>QComboBox</extends>
   <header>CustomWidgets/treebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
